v3/treasure/type9s3.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Treasure TYPE-9 Series III",
+    "vendorId": "0x5452",
+    "productId": "0x5493",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgb_matrix" ],
```

Comment:
Please enable ALL animations in QMK side if you want this. Otherwise will require a custom one. (see other examples in the repo (keychron or others))

Reply:
This is noted, custom menu has been added